app.js

```diff
@@ -0,0 +1,131 @@
+// Prerequisite data SIP and Access Token(Hard Coded)
+const myAccessToken = "ODcwYWE1MDQtODc5ZC00ZDhiLWI5ODYtZTk0ZGUyNTA4YzM2MTIwNmIzOTQtZDhm_PF84_1eb65fdf-9643-417f-9974-ad72cae0e10f";
+const sip = "pasoma.cisco@webex.com";
+
+document.addEventListener('DOMContentLoaded', () => {
+    const ongoingMeetingListElement = document.getElementById('ongoing-meeting-list');
+    const upcomingMeetingListElement = document.getElementById('upcoming-meeting-list');
+    const meetingDetailsElement = document.getElementById('meeting-details');
+    
+    const webhookApiUrl = 'https://webexapis.com/v1/meetings';
+    
+    async function fetchMeetings() {

```

Comment:
It is better to have all these function definition out of DOMContentLoaded event and just use this event to call `fetchMeetings` method.